src/main/kotlin/com/team4099/robot2020/config/Constants.kt

```diff
@@ -105,6 +105,16 @@ object Constants {
         }
     }
 
+    object Shooter {
+        const val MASTER_SPARKMAX_ID = 0
+        const val SLAVE_SPARKMAX_ID = 1
+
+        val SHOOTER_PID = PIDGains(0, 1.0, 1.0, 1.0, 0)
+
+        const val targetSpeed = 0.0
```

Comment:
should capitalize these like other constants

Reply:
the constructor for PIDGains changed so you should probably rebase with develop